extensions/cpp/package.json

```diff
@@ -50,6 +50,13 @@
           "cpp"
         ],
         "configuration": "./language-configuration.json"
+      },
+      {
+        "id": "cuda-cpp",
+        "extensions": [
+          ".cu",
+          ".cuh"
+        ]
```

Comment:
Is an 'alias' section needed?

```
        "aliases": [
          "CUDA"
        ],
```

I believe that would cause "CUDA" to be displayed in language ID selection menus instead the raw language ID.

Reply:
+1 to this.

Reply:
Thanks for pointing this out Colen. Added.

Reply:
Sorry I just saw Alex's comment. Thank you both.